lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
Ohhhh - the PID thing doesn't happen unless we'er using tunnels, so that's why...

but the current code works flawlessly. Let's see what happens when I use your code? Hmmmm, that works as well. So OK, this code seems to work, I'm OK with that... I'm curious though where you see it not working and in what way? Can you swing by so I can make sure we're not missing something?